azurelinuxagent/pa/provision/cloudinit.py

```diff
@@ -82,7 +82,11 @@ def wait_for_ovfenv(self, max_retry=360, sleep_time=5):
         for retry in range(0, max_retry):
             if os.path.isfile(ovf_file_path):
                 try:
-                    OvfEnv(fileutil.read_file(ovf_file_path))
+                    ovf_env = OvfEnv(fileutil.read_file(ovf_file_path))
+                    self.report_event(message="{0}".format(ovf_env.provision_guest_agent),
```

Comment:
Rather than using `"{0}".format(blah)` this way, just convert blah to a string, i.e.
```self.report_event(message=str(ovf_env.provision_guest_agent),```